pages/docs/viewer.mdx

```diff
@@ -320,3 +332,39 @@ return (
   />
 );
 ```
+
+---
+
+### Custom displays
+
+If you want to extend the Viewer's default display elements for a Canvas item (OpenSeadragon for images and HTML Video Player for audio/video), pass in "Custom Display" React component(s) using the Viewer's `customDisplays` prop with the following shape:
+
+```tsx
+import MyCustomDisplay from './MyCustomDisplay';
+import AnotherCustomDisplay from './AnotherCustomDisplay';
+
+type CustomDisplay = {
+  component: React.ElementType;
+  target: {
+    canvasId: string[];
+    paintingFormat: string[]; // "application/pdf" or "application/epub+zip"
```

Comment:
This declares `paintingFormat` as an array of strings, but in the below example, it is a string. I think an array is solid and will provide the most flexibility.